sample/src/main/java/com/airbnb/android/showkasesample/BasicChip.kt

```diff
@@ -44,3 +47,24 @@ fun BasicChipYellowPreview() {
         modifier = Modifier.background(color = Color.Yellow)
     )
 }
+
+@Preview(name = "Basic Chip Blue Light", group = "Chips", uiMode = UI_MODE_NIGHT_NO)
+@Preview(name = "Basic Chip Blue Dark", group = "Chips", uiMode = UI_MODE_NIGHT_YES)
+@Composable
+fun BasicChipBluePreview() {
+    BasicChip(
+        text = "Chip Component",
+        modifier = Modifier.background(color = Color.Blue)
+    )
+}
+
+
+@ShowkaseComposable(name = "Basic Chip Grey Light", group = "Chips")
```

Comment:
Since the name and group are identical with the preview ones, I feel like these composable should be getting deduped 🤔 Is that not what you observed? 

You should only see 2 composables instead of 4 in the browser

Reply:
I might be mistaken here, but I think they have different names from the Preview ones above at least. They have `Basic Chip Blue Light/Dark` and these have `Basic Chip Grey Light/Dark` 🤔 Then I think 4 components should be correct.  I could add a test case to see that they group as well if they have same name :) Wdyt? :)

Reply:
@oas004 Clearly I'm getting old. Sorry I misread that. You are correct!